KeePassNatMsg/OptionsForm.cs

```diff
@@ -64,7 +73,8 @@ private void okButton_Click(object sender, EventArgs e)
             _config.ReturnStringFieldsWithKphOnly = returnStringFieldsWithKphOnlyCheckBox.Checked;
             _config.SortResultByUsername = SortByUsernameRadioButton.Checked;
             _config.OverrideKeePassXcVersion = txtKPXCVerOverride.Text;
-            if (_restartRequired)
+			_config.ConnectionDatabaseHash = (comboBoxDatabases.SelectedItem as dynamic).DatabaseHash;
```

Comment:
i get an error when blanking out the combo box. this appears to fix it:

```suggestion
			_config.ConnectionDatabaseHash = (comboBoxDatabases.SelectedItem as dynamic)?.DatabaseHash;
```

Reply:
Thanks - my bad, haven't tested it apparently with not selecting a connection database...